templates/taxi/manufacturer_list.html

```diff
@@ -1,7 +1,9 @@
 {% extends "base.html" %}
```

Comment:
It seems that you use a 4-space indentation here. It should be changed to 2 spaces according to MA style guide. There's a [guide](https://github.com/mate-academy/py-task-guideline/blob/main/html_settings/README.MD) to help change the settings.